### ISSUE TITLE
Feature request: Specify argument names while calling macro

### ISSUE THREAD

**Issue by iredmail** [NikolaLohinski/gonja#5]
Let's say we have a macro like this:

```
{% macro greetings(to, from=simple.name, name2="name2") %} ... {% endmacro %}
```
Currently we can only call it like this (copied from [`template_tests/macro.tpl`](https://github.com/NikolaLohinski/gonja/blob/master/testData/macro.tpl):
```
{{ greetings('') }}
{{ greetings(10) }}
{{ greetings("john") }}
{{ greetings("john", "michelle") }}
{{ greetings("john", "michelle", "johann") }}
```

It should be more useful to call it with argument names like Python functions, especially when the macro has more arguments:
```
{{ greetings(to="john") }}
{{ greetings("john", from="michelle") }}
{{ greetings("john", name2="michelle", from="johann") }}  // `name2` and `from` in different order
{{ greetings("john", name2="michelle") }}  // 2nd argument `from` is missing and default value `simple.name` is assigned.
```

We're trying to port some Jinja2 macros, but macros with multiple arguments is a nightmare. This is the only issue so far while porting Jinja2 macros to gonja. Tried pongo2 too, same case.

**Comment by NikolaLohinski** [comment]
Hey @iredmail I got back to your example and figured out what was not working as you wanted it to. Only specifying keyword arguments when calling a macro was not working as expected.

It is now fixed in [v1.5.2](https://github.com/NikolaLohinski/gonja/tree/v1.5.2). Let me know if that works for you and feel free to close the ticket

cheers

**Comment by NikolaLohinski** [comment]
closing as it's now fixed. Fill free to reopen if you run into any issues with the fixes.
